feature/flags.go

```diff
@@ -48,7 +48,10 @@ const OldPresence = "old-presence"
 const LegacyLeases = "legacy-leases"
 
 // Generations will allow for model generation functionality to be used.
-const Generations = "generations"
+const Generations = "branches"
+
+// AltGenerations is the old flag name used for generations feature
+const AltGenerations = "generations"
```

Comment:
I'm going to be really picky here and rename `const Generations` to `const Branches` and then add a good comment about it. Otherwise future us, will be very confused about this!

Reply:
no worries, makes sense to me, will change =D